### PR TITLE
updating bug bounty hall of fame for Q1 2015 and Q4 2014

### PR DESCRIPTION
**Pull request by rforbes**
This update just added some data to the html page for

### CODE REVIEW

File: bin/sync_all [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-cd $(readlink --canonicalize --no-newline ${0%/*})/..
```
Comment (flodolo) [inline bin/sync_all]: You don't want this in your PR ;-)

Comment (rforbes) [inline bin/sync_all]: dang it, you are right. that was an error.  I am sorry, I am new to this git thing.  What should I do?

**Comment by pmclanahan** [comment]
Looks good and seems to work well. r+ from me. maybe @craigcook should double check the HTML though.

**Comment by craigcook** [comment]
Looks good to me!

**Pull request merged**